src/seedu/addressbook/parser/Parser.java

```diff
@@ -94,7 +96,11 @@ public Command parseCommand(String userInput) {
         case ViewAllCommand.COMMAND_WORD:
             return prepareViewAll(arguments);
 
-        case ExitCommand.COMMAND_WORD:
+            case HistoryCommand.COMMAND_WORD:
+                return new HistoryCommand(prevHistory);
+
+
```

Comment:
There is an indentation problem. In addition, there seems to be an extra blank line.